plugins/kotlin-dataframe/tests/org/jetbrains/kotlin/fir/dataframe/AbstractDataFrameBlackBoxCodegenTest.kt

```diff
@@ -65,5 +66,15 @@ open class AbstractDataFrameBlackBoxCodegenTest : AbstractFirLightTreeBlackBoxCo
     override fun createKotlinStandardLibrariesPathProvider(): KotlinStandardLibrariesPathProvider {
         return EnvironmentBasedStandardLibrariesPathProvider
     }
+
+    class SourceProvider(testServices: TestServices) : AdditionalSourceProvider(testServices) {
+        companion object {
+            const val COMMON_SOURCE_PATH = "testData/common.kt"
```

Comment:
what is containing in this file? could you explain or add a comment

Reply:
I'll rename SourceProvider -> TestUtilsSourceProvider and testData/common.kt to testData/testUtils.kt. Ok?

Reply:
Makes it possible to call compareSchemas from any test